Kconfig.zephyr

```diff
@@ -490,6 +491,12 @@ config SIZE_OPTIMIZATIONS
 	  Compiler optimizations will be set to -Os independently of other
 	  options.
 
+config MIN_SIZE_OPTIMIZATIONS
```

Comment:
Please let me know if anyone has better naming suggestions here--I don't love "min size optimizations" but I'm struggling to come up with a better name.

Reply:
I would use the same prefix as the existing `SIZE_OPTIMIZATIONS` Kconfig, and use the same "aggressive" terminology from the compile flag, i.e.: `SIZE_OPTIMIZATIONS_AGGRESSIVE`

Reply:
I like this, thanks for the suggestion! Made this change in the most recent version.